test.lua

```diff
@@ -142,14 +142,6 @@ local function init_box()
     }
     local index_type = arg[1] or os.getenv('INDEX_TYPE') or 'TREE'
     local space_type = arg[2] or os.getenv('SPACE_TYPE') or 'memtx'
-    if space_type == 'vinyl' then
-        space_type = nil
-        if box.error.VINYL ~= nil then
-            space_type = 'vinyl'
-        else
-            os.exit(0)
-        end
-    end
```

Comment:
Please extend a commit message with an info about this change

Reply:
Done.